src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+type State = {
+  time: Date;
+};
+
+type Props = {
+  name: number;
+};
+
+class Clock extends React.Component<Props, State> {
+  state = {
+    time: new Date(),
+  };
+
+  timer = setInterval(() => {
+    this.state.time.toLocaleTimeString();
+    this.setState({ time: new Date() });
+  }, 1000);
+
+  componentDidMount() {
+    return this.timer;
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  }
+
+  render() {
+    const { name } = this.props;
+    const { time } = this.state;
+
+    return (
+      <>
```

Comment:
You don't really need to use react fragment here. You already have `div` tag like wrapper